docs/UserGuide.md

```diff
@@ -31,104 +31,238 @@
+  Returns Any tasks containing the tag `CS2103` or `cs2103` but not `CS2103T` or `CS2103 Project`.    
+    
+Finding all tasks which has deadlines falling within certain dates    
+* Finds tasks which has DEADLINEs falling between the specified range.<br>    
+* Format: `find date STARTING_DATE ENDING_DATE`    
+    
+> Date must be in the form `d MMM yyyy`, `d-MM-yyyy` or `d/MM/yyyy`
+    
+Examples:   
+`find date 1 Oct 2016 31 Oct 2016`   
+  Returns Any tasks which has DEADLINEs falling in between 1st October 2016 and 31st October 2016.<br>
+`find date 1-11-2016 31-12-2016`   
+  Returns Any tasks which has DEADLINEs falling in between 1st November 2016 and 31st December 2016.
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+#### Deleting a task: `delete`
+Deletes the selected task from Lifekeeper. Irreversible.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task with `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+  
 
 Examples: 
 * `list`<br>
+  `delete 1,3`<br>
+  Deletes the 1st and 3rd task in the Lifekeeper task list.
+* `find Dinner`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Selects the 2nd task in the results of the `find` command and then deletes it.
+
+#### Select: `select`
+Category
+Selects the category identified by the index number used in the last task listing.<br>
+Format: `select INDEX`
+
+> Selects the category at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
+
+Examples: 
+* `list`<br>
+  `select 2`<br>
+  Selects the 2nd category in category list.
+
+Task
+Selects the task identified by the index number used in the last task listing.<br>
 Format: `select INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
+> Selects the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
+
 Examples: 
 * `list`<br>
   `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
+  Selects the 2nd task in Lifekeeper.
+* `find Dinner` <br> 
   `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  Selects the 1st task in the results of the `find` command.
+* `list category` <br> 
+  `select 3`<br>
+  `select 1`<br>
+  Selects the 3rd category in the list and then select the 1st task shown in the category..
+
+#### Marking a task as done: `done`
```

Comment:
what happens to the task on marking it as done? is it saved? if so where? can the user query such marked tasks?
is it shown in the list?